packages/cli/src/cmds/beacon/options.ts

```diff
@@ -57,11 +58,19 @@ export const beaconExtraOptions: CliCommandOptions<BeaconExtraArgs> = {
   },
 
   checkpointState: {
-    description: "Set a checkpoint state to start syncing from",
+    description: "Checkpoint state file path to start syncing from",
     type: "string",
     group: "weak subjectivity",
   },
 
+  lastPersistedCheckpointState: {
+    description: "Set the last persisted checkpoint state to start syncing from",
```

Comment:
just to get this right, what you mean here is "use last persisted checkpoint state"? I understand how it works from reviewing the code but we should consider rephrasing the flag / description a bit

Reply:
yes, will fix it 👍